qemu/tests/numa_memdev_mlock.py

```diff
@@ -22,8 +22,15 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
-    error_context.context("Check host's numa node(s) amount!", logging.info)
-    numa_memdev_options.check_host_numa_node_amount(test)
+    error_context.context("Check host's numa node(s)!", logging.info)
+    node_list_size_not_zero = numa_memdev_options.check_host_numa_node(test)
```

Comment:
why not put line 27 and 28 to the if block in line 30?

Reply:
I will update it.